driver/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <fstream>
+#include "reader.h"
+using namespace reader;
+int main() {
+
+    read_config("C:\\Users\\Hanko\\Documents\\Assignment3\\config.txt");
+    return 0;
+}
+
+int size_finder()
+{
+    int count = 0;
+    string line;
+    ifstream myFile("C:\\Users\\Hanko\\Documents\\Assignment3\\config.txt");
+    while(getline(myFile,line))
+    {
+        count++;
+    }
+    return count;
+}
+
+string* read_config(string config_file_path) {
```

Comment:
The reason I was talking about two projects is this function definition should be in the lib project's cpp file.
Does that make sense?